code/api/src/main/scala/com/avast/datadog4s/api/metric/Timer.scala

```diff
@@ -1,11 +1,29 @@
 package com.avast.datadog4s.api.metric
 
-import java.time.Duration
-
 import com.avast.datadog4s.api.Tag
 
+import java.time.Duration
+import java.util.concurrent.TimeUnit
+
 trait Timer[F[_]] {
   def time[A](f: F[A], tags: Tag*): F[A]
 
-  def record(duration: Duration, tags: Tag*): F[Unit]
+  def record(duration: Duration, tags: Tag*): F[Unit] = recordT[Duration](duration, tags: _*)
```

Comment:
Do we really want to keep this? We are going to introduce binary incompatibility anyway. We can keep only `def record[T](value: T ...` which should be source compatible for the most of times.

Reply:
good point